assets/src/edit-story/components/keyboardShortcutsMenu/keyboardShortcutList.js

```diff
@@ -95,73 +155,186 @@ const shortcuts = {
+              <kbd>{'C'}</kbd>
+            </kbd>
+          ),
         },
         {
           label: __('Cut', 'web-stories'),
-          shortcut: [cmdOrCtrl, 'X'],
+          shortcut: (
+            <kbd>
+              <kbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</kbd>
+              <kbd>{'X'}</kbd>
+            </kbd>
+          ),
         },
         {
           label: __('Duplicate', 'web-stories'),
-          shortcut: [cmdOrCtrl, 'D'],
+          shortcut: (
+            <kbd>
+              <kbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</kbd>
+              <kbd>{'D'}</kbd>
+            </kbd>
+          ),
         },
         {
           label: __('Paste', 'web-stories'),
-          shortcut: [cmdOrCtrl, 'V'],
+          shortcut: (
+            <kbd>
+              <kbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</kbd>
+              <kbd>{'V'}</kbd>
+            </kbd>
+          ),
         },
         {
           label: __('Select all', 'web-stories'),
-          shortcut: [cmdOrCtrl, 'A'],
+          shortcut: (
+            <kbd>
+              <kbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</kbd>
+              <kbd>{'A'}</kbd>
+            </kbd>
+          ),
         },
         {
           label: __('Select multiple', 'web-stories'),
-          shortcut: [
-            SPECIAL_KEYS.SHIFT,
-            { label: __('+ click', 'web-stories') },
-          ],
+          shortcut: (
+            <kbd>
+              <TranslateWithMarkup
+                mapping={{
+                  kbd: <kbd />,
+                  shift: SPECIAL_KEYS.SHIFT.symbol,
+                }}
+              >
+                {sprintf(
+                  /* translators: %s: Shift key. */
+                  __(
+                    '<kbd aria-label="%s"><shift /></kbd><span> + click</span>',
+                    'web-stories'
+                  ),
+                  SPECIAL_KEYS.SHIFT.title
+                )}
+              </TranslateWithMarkup>
+            </kbd>
+          ),
         },
         {
           label: __('Move forward or back', 'web-stories'),
-          shortcut: [
-            cmdOrCtrl,
-            SPECIAL_KEYS.UP,
-            { label: __('or', 'web-stories') },
-            SPECIAL_KEYS.DOWN,
-          ],
+          shortcut: (
+            <kbd>
+              <TranslateWithMarkup
+                mapping={{
+                  kbd: <kbd />,
+                  cmdOrCtrlSymbol: cmdOrCtrl.symbol,
+                  upSymbol: SPECIAL_KEYS.UP.symbol,
+                  downSymbol: SPECIAL_KEYS.UP.symbol,
+                }}
+              >
+                {sprintf(
+                  /* translators: 1: Cmd/Ctrl key. 2: Up key. 3: Down key. */
+                  __(
+                    '<kbd aria-label="%1$s"><cmdOrCtrlSymbol /></kbd><kbd aria-label="%2$s"><upSymbol /></kbd><span> or </span><kbd aria-label="%3$s"><downSymbol /></kbd>',
```

Comment:
wow this one is crazy 🔥

Reply:
That‘s a lot of markup in a translatable string. Might be a tricky for translators. Not sure what would be best here though.

Reply:
🤔 
We've got a follow up for the general popup to be announced better for SRs, let's return to this there - I don't think we need to hold the PR up for this.